tw-spyql-starter/src/main/java/com/transferwise/common/spyql/starter/SpyqlAutoConfiguration.java

```diff
@@ -0,0 +1,10 @@
+package com.transferwise.common.spyql.starter;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@Import(SpyqlConfiguration.class)
```

Comment:
Any reason for not adding the logic directly to `SpyqlAutoConfiguration` but separating it out to dedicated config?
For some kind of non Spring Boot service?

Reply:
It was copy paste. Deleted extra `SpyqlConfiguration`.